src/planscape/conditions/raster_utils.py

```diff
@@ -57,23 +70,26 @@ def compute_condition_score_from_raster(
             "no rasters available for raster_name, %s" % (raster_name))
     with connection.cursor() as cursor:
         cursor.callproc(
-            'get_mean_condition_score',
+            'get_condition_stats',
             (RASTER_TABLE, RASTER_SCHEMA, raster_name,
              RASTER_NAME_COLUMN, RASTER_COLUMN, geo.ewkb))
         fetch = cursor.fetchone()
-        if fetch is None or len(fetch) == 0:
-            return None
-        return fetch[0]
+        if fetch is None or len(fetch) != 3:
+            return ConditionStatistics({'mean': None,
+                                        'sum': 0.0,
```

Comment:
why is sum 0.0 vs None? if that's a choice we're making, can we add a comment explaining why

Reply:
The sum across zero raster pixels is 0.
The count of zero raster pixels is 0.
The mean is undefined - sum/count is 0/0.

Reply:
can we add this/what you edited in the PR description to the function comment?